algosdk/source_map.py

```diff
@@ -0,0 +1,93 @@
+from typing import Dict, Any, List, Tuple
+
+
+class SourceMap:
+    """
+    Decodes a VLQ-encoded source mapping between PC values and TEAL source code lines.
+    Spec available here: https://sourcemaps.info/spec.html
+
+    Args:
+        source_map (dict(str, Any)): source map JSON from algod
+        delimiter (str, optional): delimiter for mappings
+    """
+
+    def __init__(self, source_map: Dict[str, Any], delimiter: str = ";"):
```

Comment:
@barnjamin For my understanding:  What motivated exposing `delimiter` for user configuration?

From my read of the spec, the delimiter is _always_ `;`.  I'm trying to gauge if a situation exists where the user supplies a source map v3 compatible mapping with different delimiter.

Aside:  _If_ the delimiter is removed here, then I think the same change ought to happen in js-algorand-sdk.